.github/workflows/ci.yml

```diff
@@ -224,6 +224,7 @@ jobs:
         with:
           # Take Go version to install from go.mod
           go-version-file: 'go.mod'
+      - run: pip install wheel
```

Comment:
Always consider someone maintaining this in a few years wondering what this is for :grin: Please add a comment here (or in the commit msg) that explains what exactly this does.

It seems it's needed for `aqt` for some reason, the tool used by the Qt install action below to install Qt. Consider opening an issue at https://github.com/jurplel/install-qt-action/issues with the CI error and context and linking the issue here (so we know we can remove this again once fixed).

Reply:
Thanks, done and opened https://github.com/jurplel/install-qt-action/issues/281